### PR TITLE
update tikv configuration about pessimistic transaction

### PR DESCRIPTION
**Pull request by youjiali1995**
Signed-off-by: youjiali1995 <zlwgx1023@gmail.com>

### What is changed, added or deleted? (Required)

update tikv configuration about pessimistic transaction

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1159,11 +1159,16 @@ import 相关的配置项。
 
 ### `wait-for-lock-timeout`
 
-+ 悲观事务在 TiKV 中等待其他事务释放锁的最长时间，单位为毫秒。若超时则会返回错误给 TiDB 并由 TiDB 重试加锁，语句最长等锁时间由 `innodb_lock_wait_timeout` 控制。
-+ 默认值：1000
```
Comment (coocood) [inline tikv-configuration-file.md:1163]: The config item is an integer, not string, why change it?

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1159,11 +1159,16 @@ import 相关的配置项。
...
-+ 最小值：1
++ 悲观事务在 TiKV 中等待其他事务释放锁的最长时间。若超时则会返回错误给 TiDB 并由 TiDB 重试加锁，语句最长等锁时间由 `innodb_lock_wait_timeout` 控制。
++ 默认值：1s
++ 最小值：1ms
 
 ### `wait-up-delay-duration`
 
-+ 悲观事务释放锁时，只会唤醒等锁事务中 start ts 最小的事务，其他事务将会延迟 `wake-up-delay-duration` 毫秒之后被唤醒。
-+ 默认值：20
++ 悲观事务释放锁时，只会唤醒等锁事务中 start ts 最小的事务，其他事务将会延迟 `wake-up-delay-duration` 之后被唤醒。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1168]:
```suggestion
+ 悲观事务释放锁时，只会唤醒等锁事务中 `start_ts` 最小的事务，其他事务将会延迟 `wake-up-delay-duration` 之后被唤醒。
```

File: tikv-configuration-file.md (hunk 1 above)
Comment (youjiali1995) [reply tikv-configuration-file.md:1163]: It's a string in v4.0, and it's backward-compatible. https://github.com/tikv/tikv/pull/7197

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1159,11 +1159,16 @@ import 相关的配置项。
...
 ### `wait-up-delay-duration`
 
-+ 悲观事务释放锁时，只会唤醒等锁事务中 start ts 最小的事务，其他事务将会延迟 `wake-up-delay-duration` 毫秒之后被唤醒。
-+ 默认值：20
++ 悲观事务释放锁时，只会唤醒等锁事务中 `start_ts` 最小的事务，其他事务将会延迟 `wake-up-delay-duration` 之后被唤醒。
++ 默认值：20ms
+
+### `pipelined`
+
++ 开启流水线式加悲观锁流程。开启该功能后，TiKV 在检测完数据满足加锁要求后就立刻通知 TiDB 执行后面的请求，并异步写入悲观锁，从而降低大部分延迟，显著提升悲观事务的性能。但有较低概率悲观锁异步写入失败，可能会导致悲观事务提交失败。
```
Comment (toutdesuite) [inline tikv-configuration-file.md:1173]:
```suggestion
+ 开启流水线式加悲观锁流程。开启该功能后，TiKV 在检测数据满足加锁要求后，立刻通知 TiDB 执行后面的请求，并异步写入悲观锁，从而降低大部分延迟，显著提升悲观事务的性能。但有较低概率出现悲观锁异步写入失败的情况，可能会导致悲观事务提交失败。
```

**Comment by coocood** [comment]
LGTM

**Pull request merged**